### ISSUE TITLE
sshuttle support?

### ISSUE THREAD

**Issue by beddari** [The-Compiler/sshtunnel#1]
autossh, ok ... what about adding an option for sshuttle? :) If motivation strikes ... I'll try!

**Comment by The-Compiler** [comment]
Oh wow, people other than me are using this! :laughing:

I'd appreciate a pull request, I'm more busy with [qutebrowser](http://www.qutebrowser.org/) nowadays.

**Comment by beddari** [comment]
Yeah after again embracing (_cough_) Fedora I wanted to try and actually do stuff with systemd, so that got me here. No promises, I'll just close this now that the idea is on the record hehe

**Comment by The-Compiler** [comment]
Reopening this as I tried sshuttle recently (I read about it but never actually tried), and now I want this as well :laughing:
